ui/app/templates/components/job-editor.hbs

```diff
@@ -51,17 +51,14 @@
       {{/if}}
     </div>
     <div class="boxed-section-body is-full-bleed">
-      <IvyCodemirror
+      <div
         data-test-editor
-        aria-label="Job definition"
-        @value={{or this.job._newDefinition this.jobSpec}}
-        @valueUpdated={{this.updateCode}}
-        @options={{hash
-          mode="javascript"
+        {{code-mirror
+          screenReaderLabel="Job definition"
+          content=(or this.job._newDefinition this.jobSpec)
```

Comment:
`blocking` `question`: `JobEditor` doesn't have a `jobSpec` property and neither of these properties are reactive, however, we call `set` on `_newDefintion` which might do the trick.

Do we know where `this.jobSpec` is coming from? Searching the codebase, this is the only use of `jobSpec`?

Reply:
This could introduce a potential breaking change and thankfully it'll be shipped in a major release. But, I'd love to chat with @lgfa29 re:  UI release process.

@lgfa29 - we have some serious re-designs for the Job Detail Overview page (this PR is unrelated to that) coming down the pipe. But, this PR is a good example of potential breaking API changes in our UI code. Should we start coordinating with the release process for these types of things?

Reply:
I don't think we need any special coordination. Since this is all set to go out with 1.4.0 all we need is for the changes to be in `main` before the release.

One thing we can do is to update `bindata_assetfs.go` in `main` so that anyone running `make dev` will see the latest UI changes before the release. This can help get more eyes and test major changes before the beta.